cms/core/models/base.py

```diff
@@ -4,12 +4,15 @@
 from django.utils.decorators import method_decorator
 from django.utils.functional import cached_property
 from django.utils.translation import gettext_lazy as _
+from wagtail.admin.panels import ObjectList, TabbedInterface
 from wagtail.models import Page
 from wagtail.query import PageQuerySet
+from wagtail.utils.decorators import cached_classmethod
 
 from cms.core.cache import get_default_cache_control_decorator
 from cms.core.query import order_by_pk_position
 
+from ...taxonomy.forms import DeDuplicateTopicsAdminForm
```

Comment:
Let's avoid anything other than local module imports.
that is:

`from .mixins import X` is fine, but `from ...taxonomy.forms import X` should be `from cms.taxonomy.forms import X`

Reply:
Addressed in https://github.com/ONSdigital/dis-wagtail/pull/104/commits/42afe32cd373a02002a575d541d2e2c31d555792